src/api/service.ts

```diff
@@ -30,8 +31,11 @@ export const getProductsWithDiscount = () => {
   return axiosClient.get<Product[]>(`/${EndPoints.Product}/discount`);
 };
 
+// export const getProductDetail = (endPoint: EndPoints, itemId: string) => {
```

Comment:
If this is not needed, don't forget to remove it)

Reply:
Oh my gosh we have a hover here... Im shoking... i`ll think what could I do